graphite/graphite.go

```diff
@@ -42,23 +60,45 @@ func (r *Request) Query(host string) (Response, error) {
 	if r.End != nil {
 		v.Add("until", fmt.Sprint(r.End.Unix()))
 	}
-	u := url.URL{
+	r.url = url.URL{
 		Scheme:   "http",
 		Host:     host,
 		Path:     "/render/",
 		RawQuery: v.Encode(),
 	}
-	resp, err := DefaultClient.Get(u.String())
+}
+
+func (r *Request) String() string {
```

Comment:
what do you mean? i call this in a few places when creating errors to display the request info to the user.